@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/aero/fn_aeroStabilator.sqf

```diff
@@ -63,12 +63,29 @@ _stabOutputTable = [
                    ,[97.74, _intStabTable select 17]  //190kts
                    ,[102.9, _intStabTable select 18]  //200kts
                    ];
-
-_theta = [_stabOutputTable, fza_sfmplus_vel2D * KNOTS_TO_MPS] call fza_fnc_linearInterp select 1;
-
+*/
```

Comment:
Mattys not a fan of these large pieces of commented out code, it can always be found in the git history

Reply:
I keep these in case i need to quick swap between implementations. I'm not a fan of having to hunt for flight model code i may need in a hurry.